### ISSUE TITLE
libpng vulnerability issue when publishing app using this library. How to upgrade the libpng in this library.

### ISSUE THREAD

**Issue by SumanPahari** [DroidNinja/Document-Scanner#3]
Please clear how to use?i get your link from [https://github.com/jhansireddy/AndroidScannerDemo/issues](url). but how to use your library.or how to solve libpng issues?

**Comment by DroidNinja** [comment]
This is not a library. Its just a sample project. Libpng is fixed with the latest opencv version. This project uses Libpng from latest opencv as you can see it is live on play store.

**Comment by SumanPahari** [comment]
yes i see it .can you tell me how to remove this issues?because i have already used https://github.com/jhansireddy/AndroidScannerDemo/issues library but libpng issues comes .how to solve this please guide . you told that in latest opencv remove issues , but how to replace and how to do this ? please help
